action_server/tests/action_server_tests/test_cli/test_help.txt

```diff
@@ -1,11 +1,11 @@
 usage: action-server [-h]
-                     {start,import,download-rcc,new,version,migrate,package,env,cloud}
+                     {start,import,download-rcc,new,version,migrate,package,env,cloud,get-sema4ai-oauth-config}
                      ...
 
 Sema4.ai Action Server (<version>)
 
 positional arguments:
-  {start,import,download-rcc,new,version,migrate,package,env,cloud}
+  {start,import,download-rcc,new,version,migrate,package,env,cloud,get-sema4ai-oauth-config}
```

Comment:
This change means that it's actually visible to the user... To make it really invisible I guess you have to check `sys.args` directly before giving it to the arguments parser instead of doing it through the arguments parser if it's really meant to be invisible to the user.

-- If it can be shown to the user that'd be ok.

Reply:
@fabioz @kariharju I did remove it ; ))